.github/workflows/check-pr.yml

```diff
@@ -27,5 +27,17 @@ jobs:
     name: Check Milestone
     runs-on: ubuntu-latest
     steps:
-      - if: github.event.pull_request.milestone == null && contains( env.LABELS, 'no milestone' ) == false
-        run: exit 1
+      - name: Check if milestone or “no milestone” label is present
+        run: exit ${{ (github.event.pull_request.milestone != null || contains(env.LABELS, 'no milestone')) && '0' || '1'}}
```

Comment:
This is much harder to read than the previous version, and I think requires more knowledge of github's expression language than is necessary for the task.

Reply:
fixed